Makefile

```diff
@@ -0,0 +1,51 @@
+NAMESPACE := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["namespace"])')
```

Comment:
Nice makefile! But do we need documentation updates for it? (Assuming we have documentation building the collection currently.)

Also, if users cannot run this directly from this dir (I made a bunch of other comments), that means they need `make` installed?

Reply:
Yes, in a way this Makefile is a wrapper around the insufficiencies of molecule.
But i consider make to be quite a common dependency, especially for developers (our target audience here).
Way more common than tox at least.
I will see whether we have a "how can i run the tests" documentation.

Reply:
> I will see whether we have a "how can i run the tests" documentation.

+1